lib/grpc/GrpcService.ts

```diff
@@ -66,6 +67,7 @@ const createSwapFailure = (swapFailure: SwapFailure) => {
   grpcSwapFailure.setPeerPubKey(swapFailure.peerPubKey);
   grpcSwapFailure.setQuantity(swapFailure.quantity);
   grpcSwapFailure.setFailureReason(SwapFailureReason[swapFailure.failureReason]);
+  grpcSwapFailure.setFailTime(Date.now());
```

Comment:
This won't give us accurate times when we use the synchronous place order calls. I'm thinking we should add a `failTime` property to the `SwapDeal` type (and probably for the db as well, although that can be a separate PR) and have the `SwapFailure` type inherit it, then assign it a value of `Date.now()` when we create the `SwapFailure` in https://github.com/ExchangeUnion/xud/blob/master/lib/orderbook/OrderBook.ts#L388. Then we can use that value here.

Reply:
I will create a new PR for it. @sangaman

Reply:
Can you assign this a value in this PR at least in the rpc layer? I was thinking of a separate PR only for the db option. Or if you think that's too much, let's remove `fail_time` altogether from this PR, I just don't think it makes sense to add it to the proto file but not assign a value to it.

Reply:
Sorry. Forgot to remove that gRPC proto field.